### DISCUSSION TITLE
Where is a list of recent commits? It seems it is gone.

### DISCUSSION THREAD

**Discussion by roe-dl** [community/community#66345] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Until yesterday there were information about commits on the dashboard. And most of the items were about commits. Now no single item is shown about commits. They are gone completely. What is the purpose of this change? It seems so stupid to me.

**Comment by Fox2Code** [comment]
The only reason I use feed is to see commits.

**Comment by rusty-snake** [comment]
Exactly, I contribue to a lot projects where that are no organized like a company but like a few devs working in their free time (I guess GH is a platform for this). They have a **"review happens on master"** practice where reviews are fixed by pushing new commits master/main. Now I have to open every single repo?

**Comment by Fox2Code** [comment]
You should all check that to get the old feed back:
https://github.com/orgs/community/discussions/66188

**Comment by ancestor-mithril** [comment]
Still no answer to this?

**Comment by VedantiBhoyar** [comment]
Check GitHub Status: Sometimes, issues like this can be due to maintenance or service disruptions on GitHub's end. Check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing incidents that might be causing this problem.

Dashboard Settings: Make sure that your dashboard settings are configured correctly. GitHub allows you to customize your dashboard to show specific information. Go to your dashboard, and at the top right, you should see an "Customize your pins" option. Check if commits are selected there.

Browser Cache: Clear your browser cache and cookies, then refresh the page. Cached data can sometimes cause display issues.

Browser Extensions: Disable any browser extensions or add-ons you might be using, as they could interfere with the display of GitHub's dashboard.

Check Repository Activity: Sometimes, the activity feed on your dashboard might be dominated by one particular repository. Check the recent activity in your repositories to ensure that commits are still happening.

GitHub Settings: Explore your GitHub account settings to see if there are any options related to the dashboard and the visibility of commit information. Make sure that your settings haven't changed unexpectedly.

**Comment by dharmendra-byte** [comment]
best

**Comment by dharmendra-byte** [comment]
it is a better
